text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Token API will provide token information by requesting `hathor-wallet-service`.
+
+# Motivation
+[motivation]: #motivation
+
+We need to retrieve token data without requesting the full node. Wallet service holds those data, so, we'll use it for now. 
```

Comment:
The motivation is to replace the full node's API so the explorer can scale as much as we want without hitting the full node direclty.